app/javascript/hooks/mutations/rooms/useUploadPresentation.jsx

```diff
@@ -0,0 +1,26 @@
+import { useMutation, useQueryClient } from 'react-query';
+import axios from 'axios';
+
+export default function useUploadPresentation(friendlyId) {
+  const queryClient = useQueryClient();
+
+  const uploadPresentation = (data) => {
+    const formData = new FormData();
+    formData.append('presentation', data);
```

Comment:
Is this still needed? Can we not just send the info in the body as we regularly do?

Reply:
For some reason it causes issue with data={}. This is something I will look into towards the end when cleaning up

Reply:
I've read that Files need a different packaging than the default JSON.stringify().